source/platforms/BitBucketServer.ts

```diff
@@ -232,8 +232,9 @@ export class BitBucketServer implements Platform {
     // https://bitbucket.org/atlassian/jiraconnect-ios/pull-requests/4/crash-groups/diff#comment-426
     // So take the PR and attack the comment, I think :D
 
+    const host = this.api.repoCredentials.host
     const prURL = this.api.getPRBasePath()
-    return issue && issue.id && `${prURL}/diff#comment-${issue.id}`
+    return issue && issue.id && `${host}${prURL}/diff#comment-${issue.id}`
```

Comment:
The `prURL` here is the API URL (`rest/api/1.0/...`) which will link to the JSON blob of the diff, rather than the web UI.

Reply:
Aww, well, at least it's not crashing - I had no way to test 💯